### PR TITLE
clevis: add pytest

### PR DESCRIPTION
**Pull request by lindenb1**

### CODE REVIEW

File: roles/clevis/defaults/main.yml [hunk 1]
```diff
@@ -4,4 +4,5 @@ clevis_tang_server: 127.0.0.1
 clevis_tang_server_protocol: 80
 clevis_tang_url: "http://{{ clevis_tang_server }}:{{ clevis_tang_server_protocol }}"
 clevis_initial_luks_password: password
+clevis_luks_slot: 1
```
Comment (gndrmnn) [inline roles/clevis/defaults/main.yml:7]: Should be ``2`` to remain consistent with previous role implementation, and ``1`` as overwrite in the test

File: molecule/delegated/tests/clevis/redhat.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+testinfra_runner, testinfra_hosts = get_ansible()
+
+
+def check_ansible_os_family(host):
+    if get_variable(host, "ansible_os_family", True) != "RedHat":
+        pytest.skip("ansible_os_family mismatch")
+
+
+def test_clevis_packages(host):
```
Comment (gndrmnn) [inline molecule/delegated/tests/clevis/redhat.py:13]: ``test_clevis_packages`` could be moved to main.py

File: roles/clevis/defaults/main.yml (hunk 1 above)
Comment (sbstnnmnn) [reply roles/clevis/defaults/main.yml]: Setting the variable clevis_luks_slot to 2 is also compatible with our test suite.

File: molecule/delegated/tests/clevis/redhat.py (hunk 2 above)
Comment (sbstnnmnn) [reply molecule/delegated/tests/clevis/redhat.py]: Done

**Review by gndrmnn — APPROVED** [review]

**Pull request merged**